src/Components/Main/Pages/UsersPage/User/User.module.scss

```diff
@@ -0,0 +1,42 @@
+.user {
+  width: 50%;
+  display: flex;
+  margin: $marginMini auto $marginMini auto;
+  padding: $padding;
+  border: 1px solid $active;
+  border-radius: $radius;
+  .userMainBlock {
+    width: 40%;
+    border-right: 1px solid $content;
+    .userPhotoBlock {
+      width: 50%;
+      &:not(:last-child) {
+        margin: 0 auto $marginMini auto;
+      }
+      .userPhoto {
+        width: 100%;
+        border: 1px solid $content;
+        border-radius: $radius;
+      }
+    }
+  }
+  .userInfoBlock {
+    width: 60%;
+    .title {
+      @extend %mini-title;
+      margin-bottom: $marginMini;
+    }
+    .userInfo {
+      margin-left: $marginMaxi;
```

Comment:
Just wondering, is it correct to call it "Maxi", should it be just "Max"? =)

Reply:
Done!